sklearn/tests/test_grid_search.py

```diff
@@ -768,3 +768,53 @@ def test_parameters_sampler_replacement():
     sampler = ParameterSampler(params_distribution, n_iter=7)
     samples = list(sampler)
     assert_equal(len(samples), 7)
+
+
+_atleast_py34 = (sys.version_info.major == 3 and
+                 sys.version_info.minor >= 4)
+
+
+@np.testing.decorators.skipif(not _atleast_py34)
+def test_grid_search_backend():
+    # copied from test_grid_search
+    from multiprocessing import pool
+    backend = pool.get_context(method="forkserver")
```

Comment:
I thought I had documented it in the joblib docstrings but apparently it's not the case...